### PR TITLE
Create Rule “how-to-decide-what-to-test/rule”

### PR DESCRIPTION
**Pull request by leehawkinsssw**
Automatically generated by Netlify CMS

### CODE REVIEW

File: rules/how-to-decide-what-to-test/rule.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+* **U**sers' Desires. We believe that the system should be consistent with ideas about what reasonable users might want. 
+* **P**roduct. We expect each element of the system (or product) to be consistent with comparable elements in the same system.
+* **P**urpose. We expect the system to be consistent with the explicit and implicit uses to which people might put it.
+* **S**tatutes. We expect a system to be consistent with laws or regulations that are relevant to the product or its use.
+
+As an example of using the "Product" part of this heuristic, note that in Trello, there is an inconsistency in the way fields on cards can be edited. Clicking in the Description or Tasks fields puts the field straight into Edit mode, whereas to edit a Comment requires an explicit click on the "Edit" action. This is a case of the Trello product being inconsistent with itself in terms of editing fields on a card.
+
+### Further reading
+
+[Cultivate Your Credibility With Oracles And Heuristics](https://www.ministryoftesting.com/dojo/lessons/cultivate-your-credibility-with-oracles-and-heuristics) by Lee Hawkins
```
Comment (tiagov8) [inline rules/how-to-decide-what-to-test/rule.md:66]: I would make this as a unordered list

Comment (leehawkinsssw) [reply rules/how-to-decide-what-to-test/rule.md:66]: For a single item? Happy for you to change it if that's the standard, but feels a bit odd to me

File: rules/how-to-decide-what-to-test/rule.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  - title: Lee Hawkins
+    url: https://www.ssw.com.au/people/lee-hawkins
+related:
+  - how-you-know-found-problem
+  - what-is-exploratory-testing
+  - risk-based-testing
+created: 2022-12-08T00:05:15.114Z
+guid: 22353d13-869b-48b5-b775-a8a872167c24
+---
+While user stories should have good [acceptance criteria](https://ssw.com.au/rules/acceptance-criteria), checking that these criteria are met is really just the starting point before engaging in deeper testing.
```
[the remark this answers is not in the document]
Comment (leehawkinsssw) [reply rules/how-to-decide-what-to-test/rule.md:15]: so the links should just be (/acceptance-criteria)? Is there a reason internal links should be relative?

File: rules/how-to-decide-what-to-test/rule.md (hunk 1 above)
Comment (tiagov8) [reply rules/how-to-decide-what-to-test/rule.md]: > For a single item? Happy for you to change it if that's the standard, but feels a bit odd to me

Yes, for consistency. I have made that change.

BTW - I wouldn't capitalize "Your", "With", and "And" as per https://www.grammarly.com/blog/capitalization-in-the-titles/ 
Thoughts?

Comment (leehawkinsssw) [reply rules/how-to-decide-what-to-test/rule.md]: I try to match the name exactly with whatever I'm linking to, as I've done in this case. In naming my own stuff, I follow something like the Grammarly recommendation.

File: rules/how-to-decide-what-to-test/rule.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (leehawkinsssw) [reply rules/how-to-decide-what-to-test/rule.md]: I guess not, stylistically I've found it easy to have full URLs so I know exactly where they're going, but no issue with changing to internal style for the rules if it's preferred.

[the remark this answers is not in the document]
Comment (leehawkinsssw) [reply rules/how-to-decide-what-to-test/rule.md]: I don't know how to make these changes, @tiagov8

[the remark this answers is not in the document]
Comment (leehawkinsssw) [reply rules/how-to-decide-what-to-test/rule.md]: I think I've made the URL changes now, @tiagov8

**Review by AlexRothwell — APPROVED** [review]
LGTM
Just make sure @tiagov8 is happy with the resolution of his comments

**Review by tiagov8 — APPROVED** [review]

File: rules/how-to-decide-what-to-test/rule.md (hunk 1 above)
Comment (tiagov8) [reply rules/how-to-decide-what-to-test/rule.md]: I see what you mean, but I think we should fix it anyways, so I've update the file. Won't make a difference to the reference but will for us (consistency)

**Pull request merged**